LanguageFeatures/Inline-classes/static_analysis_inline_class_A07_t01.dart

```diff
@@ -0,0 +1,82 @@
+// Copyright (c) 2023, the Dart project authors.  Please see the AUTHORS file
+// for details. All rights reserved. Use of this source code is governed by a
+// BSD-style license that can be found in the LICENSE file.
+
+/// @assertion A compile-time error occurs if an inline type declares a member
+/// whose name is declared by Object as well.
+///
+/// @description Checks that it is a compile-time error if an inline type
+/// declares a member whose name is declared by `Object` as well.
+/// @author sgrekhov22@gmail.com
+
+// SharedOptions=--enable-experiment=inline-class
+
+inline class V1 {
+  final int id;
+  V1(this.id);
+
+  int get hashCode => id.hashCode;
+//        ^^^^^^^^
+// [analyzer] unspecified
+// [cfe] unspecified
+
+  Type get runtimeType => V1;
+//         ^^^^^^^^^^^
+// [analyzer] unspecified
+// [cfe] unspecified
+
+  dynamic noSuchMethod(Invocation invocation) { return null;}
```

Comment:
Nit -
```suggestion
  dynamic noSuchMethod(Invocation invocation) { return null; }
```
Just guessing that `dart format` prefers that space. Maybe `=> null;` even, to avoid a 3 line format.

Reply:
Yes, `=> null` is better. I'll change it